x-pack/plugins/infra/public/components/logging/log_analysis_setup/setup_flyout/module_list_card.tsx

```diff
@@ -33,7 +48,15 @@ export const LogAnalysisModuleListCard: React.FC<{
         />
       </CreateJobButton>
     ) : (
-      <RecreateJobButton hasSetupCapabilities={hasSetupCapabilities} onClick={onViewSetup} />
+      <>
+        <RecreateJobButton hasSetupCapabilities={hasSetupCapabilities} onClick={onViewSetup} />
+        <EuiButtonEmpty {...viewInMlLinkProps}>
+          <FormattedMessage
+            id="xpack.infra.logs.analysis.viewInMlButtonLabel"
+            defaultMessage="View in Machine Learning"
+          />
+        </EuiButtonEmpty>
+      </>
```

Comment:
The vertical rhythm is a bit weird. Do we want to insert a `<EuiSpacer size="xs" />` above it like shown in the EUI card docs?

Reply:
I can add it. I could not find the documentation that you mentioned though :/

Reply:
I was referring to the double-row example code on the `<EuiCard>` documentation page:

```jsx
      <EuiCard
        icon={<EuiIcon size="xxl" type="savedObjectsApp" />}
        title="Save Objects"
        description="Example of a short card description."
        footer={
          <div>
            <EuiButton aria-label="Go to Save Objects">Go for it</EuiButton>
            <EuiSpacer size="xs" />
            <EuiText size="s">
              <p>
                Or try <EuiLink href="http://google.com">this</EuiLink>
              </p>
            </EuiText>
          </div>
        }
      />
```